File: 1916-FindCenterOfStarGraph/1916-FindCenterOfStarGraph.py
# Last updated: 12/28/2025, 4:52:34 PM
class Solution:
    def findCenter(self, edges: List[List[int]]) -> int:
        ans=0
        for i in range(0,2):
            for i in edges[0]:
                if i in edges[1]:
                    ans=i
        return ans

File: 1916-FindCenterOfStarGraph/1916-FindCenterOfStarGraph.java
// Last updated: 12/28/2025, 4:52:33 PM
class Solution {
    public int findCenter(int[][] edges) {
        int p = edges[0][0], q = edges[0][1];
        int r = edges[1][0], s = edges[1][1];
        return p == r || p == s ? p : q;
    }
}
